lib/templates/tailwind.config.js

```diff
@@ -7,5 +7,16 @@
 module.exports = {
   theme: {},
   variants: {},
-  plugins: []
+  plugins: [],
+  purge: {
+    // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
+    enabled: process.env.NODE_ENV === 'production',
```

Comment:
Should be enabled in prod by default (See the paragraph before https://tailwindcss.com/docs/controlling-file-size/#enabling-manually)

Reply:
I know, the idea is to show that they can update it here :)

Reply:
Ah, sure! That's also an option. I personally would prefer the option with less boilerplate as the link to the docs should provide more than enough info 😋